packages/scene-graph-mediator/client/src/cli.ts

```diff
@@ -15,8 +14,14 @@ export default function cli(): void {
   try {
     args = parseArgs();
   } catch (e) {
-    console.log(e);
-    console.log(CliHelptext);
```

Comment:
There seems to be no opportunity for usage to be displayed.

Reply:
`commander` shows instruction with `--help` option.
```
$ ./bin/cli.js --help
```